app/views/ui-components/v1/cards/_summary.html.erb

```diff
@@ -0,0 +1,295 @@
+<% if @hbx_enrollment.is_shop? %>
+  <% member_enrollment = @member_group.group_enrollment %>
+  <% @plan = member_enrollment.product %>
+<% else %>
+  <% @plan = @member_group %>
+<% end %>
+
+<% unless @hbx_enrollment.shopping? %>
+  <% back_btn_text = request.referrer&.include?('enrollment_history') ? l10n("back_to_enrollments") : l10n("back_to_my_account") %>
+  <%= h(link_to back_btn_text.to_s, request.referrer, class: 'btn btn-default all-plans mb-2') %>
+<% end %>
+
+<% if @bs4 %>
+  <div class="enrollment-details">
+    <% if qhp.present? %>
+      <h1><%= @hbx_enrollment.coverage_year %> <%= @hbx_enrollment&.product&.kind.to_s.titleize %> <%= l10n('coverage').titleize %></h1>
+
+      <%= render partial: "insured/families/enrollment_refactored", :collection => [@hbx_enrollment], :as => :hbx_enrollment, locals: { read_only: false } %>
+
+      <% unless @hbx_enrollment.shopping? %>
+        <% back_btn_text = request.referrer&.include?('enrollment_history') ? l10n("back_to_enrollments") : l10n("back_to_my_account") %>
+        <%= h(link_to back_btn_text.to_s, request.referrer, class: 'btn btn-default all-plans mb-2') %>
+      <% end %>
+
+      <% if display_enr_summary_is_enabled(@hbx_enrollment) %>
+        <h2 data-cuke="enrollment_detail"><%= l10n('enrollment.details.header') %></h2>
+        <dl>
+          <dt><%= l10n('enrollment.effective_on') %></dt>
+          <dd><%= @hbx_enrollment&.effective_on %></dd>
+
+          <dt><%= l10n('enrollment.transitions') %></dt>
+          <dd class="pre-line"><%= all_transitions(@hbx_enrollment) %></dd>
+
+          <dt><%= l10n('product_hios_id') %></dt>
+          <dd><%= @hbx_enrollment&.product&.hios_id || l10n('not_available') %></dd>
+
+          <dt><%= l10n('rating_area.exchange_provided_code') %></dt>
+          <dd><%= @hbx_enrollment&.rating_area&.exchange_provided_code || l10n('not_available') %></dd>
+
+          <dt><%= l10n('service_area') %></dt>
+          <dd><%= @hbx_enrollment&.product&.service_area&.issuer_provided_code || l10n('not_available') %></dd>
+        </dl>
+
+        <h2 data-cuke="enrollment_member_detail"><%= l10n('enrollment.members.header') %></h2>
+        <% @hbx_enrollment.hbx_enrollment_members.each do |member| %>
+          <h3><%= member.full_name.titleize %></h3>
+          <dl>
+            <dt><%= l10n('enrollment_member.coverage_state_date') %></dt>
+            <dd><%= member.coverage_start_on %></dd>
+          </dl>
+        <% end %>
+      <% end %>
+
+      <h2><%= l10n("enrollment_member.enrollment_services") %></h2>
+      <table>
+        <thead>
+          <tr>
+            <th><%= l10n("enrollment_member.services") %></th>
+            <th><%= l10n("enrollment_member.your_cost") %></th>
+            <th><%= l10n("enrollment_member.in_network") %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <%= render partial: "shared/service_visit_details", locals: { qhp: qhp } %>
```

Comment:
does this partial need to be uplifted for bs4?

Reply:
This is a really tiny partial (<10lines) that i didn't mess with because it was looking just fine w/ BS4.